magento/adobe-stock-integration
# [Refactoring] Move MediaGallery related modules to Magento core

### Description (*)
Review fixes after https://github.com/magento-engcom/magento2ce/pull/3943 

### Fixed Issues (if relevant)

1. Related to magento/adobe-stock-integration#1578

### Manual testing scenarios (*)
Need to check following areas
1. Check messaging for images deleting + batch
2. Media gallery and media content synchronization
3. Make sure Enhanced media gallery is disabled by default.
